catalog-rest-service/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -0,0 +1,160 @@
+import classNames from 'classnames';
+import { capitalize } from 'lodash';
+import React, { useState } from 'react';
+import { TITLE_FOR_NON_ADMIN_ACTION } from '../../constants/constants';
+import { useAuth } from '../../hooks/authHooks';
+import { isEven } from '../../utils/CommonUtils';
+import { Button } from '../buttons/Button/Button';
+import NonAdminAction from '../common/non-admin-action/NonAdminAction';
+import Searchbar from '../common/searchbar/Searchbar';
+import PageContainer from '../containers/PageContainer';
+
+const MOCK_INGESTIONS = [
+  {
+    name: 'BigQuery Ingest',
+    type: 'Usage',
+    service: 'BigQuery',
+    schedule: '1 day, 00:00:00',
+    reecentRuns: ['error', 'success', 'error', 'success', 'error'],
+    nextRun: '2021-11-19 14:00:00',
+  },
+  {
+    name: 'Snowflake Ingest',
+    type: 'Profiler',
```

Comment:
there is no type "profiler" and use lower case for type

Reply:
@harshach sure.